### PR TITLE
qa_crowbarsetup: Remove Cloud 3 and Cloud 4 support

### PR DESCRIPTION
**Pull request by toabctl**
Both are EOL.

### CODE REVIEW

**Comment by jsuchome** [comment]
+1

**Comment by nicolasbock** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -4100,13 +4091,9 @@ function onadmin_reapply_openstack_proposals()
...
-    if iscloudver 4minus ; then
-        complain 11 "This upgrade path is only supported for Cloud 5+"
-    else
-        # using the API, due to missing crowbar cli integration
-        # move nodes to upgrade mode
-        safely curl -s -X POST $crowbar_api_digest $crowbar_api/installer/upgrade/prepare.json
-    fi
+    # using the API, due to missing crowbar cli integration
+    # move nodes to upgrade mode
+    safely curl -s -X POST $crowbar_api_digest $crowbar_api/installer/upgrade/prepare.json
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: If we remove Cloud 4 support, then it means we remove the ability to upgrade from 4 to 5, which is debatable in the first place as it's a valid jenkins job we have for Cloud 5.

However, if we remove that ability, then we can drop a whole codepath related to the old way of doing upgrades.

**Comment by vuntz** [comment]
A couple of other things you can cleanup:

- `getcloudver`
- look for `GM4`, `develcloud4`
- look for `iscloudver 5plus`

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: Hm. true. I'll close it for now and leave C4 supported.

**Pull request closed without merging**